src/Fixer/Comment/CommentToPhpdocFixer.php

```diff
@@ -155,18 +155,26 @@ private function fixCommentMultiLine(Tokens $tokens, array $indices)
         $startIndex = reset($indices);
         $indent = Utils::calculateTrailingWhitespaceIndent($tokens[$startIndex - 1]);
 
-        $content = '/**'.$this->whitespacesConfig->getLineEnding();
+        $newContent = '/**'.$this->whitespacesConfig->getLineEnding();
         $count = max($indices);
 
         for ($index = $startIndex; $index <= $count; ++$index) {
             if ($tokens[$index]->isComment()) {
-                $content .= $indent.' *'.$this->getMessage($tokens[$index]->getContent()).$this->whitespacesConfig->getLineEnding();
+                $newContent .= $indent.' *'.$this->getMessage($tokens[$index]->getContent()).$this->whitespacesConfig->getLineEnding();
```

Comment:
before concat'ing the new content the `if (false !== strpos($newContent, '*/')) {` check can be done I think? allowing for an early return